Makefile

```
@@ -125,12 +131,18 @@ dev: ## Run agent executable
 	@echo "🚀 Running App"
 	$(GORUN) $(PROJECT_DIR)/$(PROJECT_FILE)
 
-run-mock-management-server: ## Run mock management server
-	@echo "🚀 Running mock management server"
-	$(GORUN) test/mock/cmd/main.go -configDirectory=$(MOCK_MANAGEMENT_PLANE_CONFIG_DIRECTORY)
+run-mock-management-grpc-server: ## Run mock management plane gRPC server
+	@echo "🚀 Running mock management plane gRPC server"
+	$(GORUN) test/mock/grpc/cmd/main.go
+
+run-mock-management-http-server: ## Run mock management HTTP server
+	@echo "🚀 Running mock management plane HTTP server"
+	$(GORUN) test/mock/http/cmd/main.go -configDirectory=$(MOCK_MANAGEMENT_PLANE_CONFIG_DIRECTORY)
 
 generate: ## Generate proto files and server and client stubs from OpenAPI specifications
 	@echo "Generating proto files"
+	@protoc --go_out=paths=source_relative:./api/grpc/ ./api/grpc/mpi/v1/*.proto --proto_path=./api/grpc/ --go-grpc_out=./api/grpc --doc_out=./${BUILD_DIR}/$(DOCS_DIR)/${PROTO_DIR}/ --doc_opt=markdown,protos.md 
```

super nit: could we please try to be consistent with using the same type of brackets, don't care which one it is.
```suggestion
	@protoc --go_out=paths=source_relative:./api/grpc/ ./api/grpc/mpi/v1/*.proto --proto_path=./api/grpc/ --go-grpc_out=./api/grpc --doc_out=./$(BUILD_DIR)/$(DOCS_DIR)/$(PROTO_DIR)/ --doc_opt=markdown,protos.md 
```
Same applies for some of the other edits in this Makefile.